pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -349,22 +339,15 @@ func setupNS(r ndctl.Region, percentage uint) error {
 		klog.V(4).Infof("MaxAvailableExtent in Region:%v is less than desired size, limit to that", r.MaxAvailableExtent())
 		canUse = r.MaxAvailableExtent()
 	}
-	// Align down to next real alignment boundary, as trying creation above it may fail.
-	canUse /= realalign
-	canUse *= realalign
-	// If less than 2GB usable, don't attempt as creation would fail
-	minsize := 2 * GB
-	if canUse >= minsize {
-		klog.V(3).Infof("Create %v-bytes fsdax-namespace", canUse)
-		_, err := r.CreateNamespace(ndctl.CreateNamespaceOpts{
-			Name:  "pmem-csi",
-			Mode:  "fsdax",
-			Size:  canUse,
-			Align: align,
-		})
-		if err != nil {
-			return fmt.Errorf("failed to create PMEM namespace with size '%d' in region '%s': %v", canUse, r.DeviceName(), err)
-		}
+	klog.V(3).Infof("Create %v-bytes fsdax-namespace", canUse)
+	_, err := r.CreateNamespace(ndctl.CreateNamespaceOpts{
+		Name:  "pmem-csi",
+		Mode:  "fsdax",
+		Size:  canUse,
+		Align: align,
```

Comment:
Why do we need to set an alignment here at all? Is there any scenario where the caller of `CreateNamespace` needs to set the alignment? If not, then let's remove `Align` from `CreateNamespaceOpts`.